smartctl, e2fsck: improve disk checking commands (#8452)
tldr-pages/tldr: pages/linux/e2fsck.md

# e2fsck

> > Check a Linux ext2/ext3/ext4 filesystem. The filesystem should be unmounted at the time the command is run.

Reviewer: I have a minor preference that we keep the original text; passive voice is more appropriate when the actor is unknown and we want to emphasize the action itself.

That said that's a non-blocker and we can move ahead with this if someone else signs off.

Author: I would still prefer "Unmount the partition first!", but you require a dot. The original is too long either way, and unnecessarily complicated in structure.

> > Check a Linux ext2/ext3/ext4 filesystem. You should unmount the partition first.

Reviewer 2: ```suggestion
> Check a Linux ext2/ext3/ext4 filesystem. The partition should be unmounted.
```

> > Check a Linux ext2/ext3/ext4 filesystem. Unmount the partition before running a command.

Reviewer 3: I prefer the prior version of this line, its a little more descriptive, and at a first glance, I thought this command would unmount it automatically, it seems like a statement rather than a command.

Author: I think the original sentence is too long. What about adding an exclamation mark at the end?
